src/test/java/it/gov/pagopa/apiconfig/util/TestUtil.java

```diff
@@ -43,11 +90,26 @@ public static Stazioni getMockStazioni() throws IOException {
     return new ObjectMapper().readValue(mockStation, Stazioni.class);
   }
 
+  public static Canali getMockChannel() throws IOException {
+    String mockChannel = readJsonFromFile("request/get_channel_ok1.json");
+    return new ObjectMapper().readValue(mockChannel, Canali.class);
+  }
+
+  public static Canali getMockChannelMapping() throws IOException {
```

Comment:
generalize this methods to avoid proliferation 


```
 public static T getMockRequest(String requestPath, Class clazz) throws IOException {
    String mockChannel = readJsonFromFile(requestPath);
    return new ObjectMapper().readValue(mockChannel, clazz);
  }
```

Reply:
I think that the `getMockXXX` method must not be removed from the class because they describe the type of resource mocked from JSON file: an invocation of the kind `getMockRequest("request/get_channel_ok2.json", Canali.class);` (used on mapping testing) will be not much clearer and does not differ much from a similar `getMockRequest("request/get_channel_ok1.json", Canali.class);` (used for generating mock models).